apps/block_scout_web/lib/block_scout_web/views/smart_contract_view.ex

```diff
@@ -6,6 +6,8 @@ defmodule BlockScoutWeb.SmartContractView do
   alias Explorer.Chain.Hash.Address, as: HashAddress
   alias Explorer.SmartContract.Helper
 
+  @tab "&nbsp;&nbsp;&nbsp;&nbsp;"
```

Comment:
?

Reply:
this module is very.. unique

Reply:
Yeah, it comes from [here](https://github.com/blockscout/blockscout/commit/643969b50576cef4335cf5130384428bb3073009) it will be removed [later](https://github.com/blockscout/blockscout/commit/fccfee176ff24129d228d5639d2b14c26ac6db19), so for simplicity I would keep it as is...